### ISSUE TITLE
Bilby (Lighthouse End-User Experience Team) - App Directory - Accessibility Staging Review

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#16432]
This will be the epic for storing accessibility findings from the [Bilby App Directory staging review](https://github.com/department-of-veterans-affairs/va.gov-team/issues/16230) on 11/19/2020.

**Comment by 1Copenut** [comment]
@ricetj I've got all my issues entered ahead of a prod launch. LMK when you're ready for me to test there as well.

**Comment by ricetj** [comment]
@1Copenut Thank you for putting all this together. We will be live around 4:00 pm est today at this url: http://www.va.gov/resources/find-apps-you-can-use

**Comment by 1Copenut** [comment]
Thank you @ricetj I'll put this on the short list of things to do. It might be Monday / Tuesday before I can finish up, but I didn't see big items in the first pass. It should be a straightforward audit.

**Comment by ricetj** [comment]
> Thank you @ricetj I'll put this on the short list of things to do. It might be Monday / Tuesday before I can finish up, but I didn't see big items in the first pass. It should be a straightforward audit.

Sounds great, thank you for all the great work you do @1Copenut !!

**Comment by 1Copenut** [comment]
@ricetj I've taken a second look in production and added a few more issue tickets. LMK if you have any questions.

**Comment by LeKeve** [comment]
@1Copenut The remaining issues are ready to be tested/verified in production.

**Comment by 1Copenut** [comment]
@ricetj I verified all of the fixes this afternoon. The next step will be for you to reach out to the VA 508 office and let them know the app is in production, is code stable, and ready for a quick test. I can send you an email address for our primary contact. Send me a DM tomorrow or the next day and we'll get that taken care of. Thank you!

**Comment by LeKeve** [comment]
@ricetj Just to fill you in. I sent an email to Linsey DeLisle at va.gov to request a 508 compliance test in production.
